liquibase-core/src/main/java/liquibase/sqlgenerator/core/InsertOrUpdateGeneratorHsql.java

```diff
@@ -54,11 +52,13 @@ protected String getElse(Database database) {
 
 	@Override
 	protected String getUpdateStatement(InsertOrUpdateStatement insertOrUpdateStatement, Database database,
-			String whereClause, SqlGeneratorChain sqlGeneratorChain) {
+										String whereClause, SqlGeneratorChain sqlGeneratorChain) throws LiquibaseException {
+
+		if (insertOrUpdateStatement.getOnlyUpdate()) {
```

Comment:
Only this code added, everything else is a revert of #1563

Reply:
Makes sense, thanks